billboardd/dbusbillboard.xml

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+    <interface name="io.thp.billboard">
+        <method name="render">
```

Comment:
Given that rendering does implicitly re-read all configs (as configs are always read as they change), that'd work, but I'm fine with `render()`